frontend/src/modules/location/templates/locationTitle.hbs

```diff
@@ -1,3 +1,31 @@
 <div class="location-title-inner">
-	{{title}}
+	{{#if breadcrumbs}}
```

Comment:
As much as I ❤️ tabs, the style guide dictates that indentation should be two spaces in this file.